openapi/components/schemas/badgeAwardItem.yaml

```diff
@@ -0,0 +1,178 @@
+title: A badgeAward item
+description: A Badge Award item
+readOnly: true
+type: object
+properties:
+  badge:
+    type: object
+    properties:
+      id:
+        $ref: ./id.yaml
+      badge:
```

Comment:
The badge details themselves shouldn't be in another level of `badge` object:
![image](https://user-images.githubusercontent.com/407053/190282475-750efff2-8c9e-498b-8f92-53267f25fb5a.png)

Reply:
fixed